proxy_agent/src/common/windows.rs

```diff
@@ -52,7 +55,11 @@ mod tests {
     #[test]
     fn get_memory_in_mb_test() {
         let memory = super::get_memory_in_mb();
-        println!("Memory in MB: {}", memory);
-        assert_ne!(0, memory, "Memory cannot be 0.");
+        match memory {
+            Ok(memory) => {
+                assert_ne!(0, memory, "Memory cannot be 0.");
+            }
+            Err(e) => panic!("Failed to get memory: {}", e),
```

Comment:
donot use panic! in test, use `assert!(false, format!("Failed to get memory: {}", e));`